kong/db/dao/plugins.lua

```diff
@@ -355,5 +355,29 @@ function Plugins:get_handlers()
   return list
 end
 
+function Plugins:execute_plugin_init()
```

Comment:
@bungle I'm trying to add support for plugins executing their own code in the `init` phase, so that plugins can do things that may contain yield which cannot be done inside a `require`. I'm adding an `execute_plugin_init` function to achieve that. How do you feel about this? Is it a proper way or any advice?

Reply:
my 2cts: in the "global patches", replace `require` with a pure-Lua version. Since this has been a recurring theme for a long time. That would solve a bunch of initialization problems due to "yield across C-boundary" issues.

Reply:
https://github.com/Kong/kong/pull/11177#pullrequestreview-1523553873
Leaving a note here: this is the discussion on the same topic. Apparently needs some works to be done to achieve that. If we can achieve that it'll be a painkiller for the restriction of not able to do any yield-able thing in module level